cmd/juju/commands/main_test.go

```diff
@@ -144,9 +144,9 @@ func (s *MainSuite) TestActualRunJujuArgOrder(c *gc.C) {
 	s.PatchEnvironment(osenv.JujuModelEnvKey, "current")
 	logpath := filepath.Join(c.MkDir(), "log")
 	tests := [][]string{
-		{"--log-file", logpath, "--debug", "controllers"}, // global flags before
-		{"controllers", "--log-file", logpath, "--debug"}, // after
-		{"--log-file", logpath, "controllers", "--debug"}, // mixed
+		{"--log-file", logpath, "--debug", "help"}, // global flags before
```

Comment:
Here we change the command that is used to test the permutation. The `controllers` command doesn't work anymore since it no longer reports success when no controllers are registered.